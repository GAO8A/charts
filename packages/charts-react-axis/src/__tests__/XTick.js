import React from 'react';
import renderer from 'react-test-renderer';
import XTick from '../XTick';

describe('XTick Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <XTick offset={250} label="Label" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
