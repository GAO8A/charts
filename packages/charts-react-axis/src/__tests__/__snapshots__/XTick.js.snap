exports[`XTick Component should render 1`] = `
<g
  className="tick"
  transform="translate(250,0)">
  <line
    stroke="#000"
    x1="0.5"
    x2="0.5"
    y2="6" />
  <text
    dy="0.71em"
    fill="#000"
    x="0.5"
    y="9">
    Label
  </text>
</g>
`;
